testsuite/cluster-test/src/main.rs

```diff
@@ -473,6 +500,56 @@ impl ClusterTestRunner {
         Ok(())
     }
 
+    pub fn perf_run(&mut self) {
+        let results = self.measure_performance().unwrap();
+        println!("{}", results.to_slack_message())
+    }
+
+    fn measure_performance(&mut self) -> failure::Result<SuiteReport> {
+        info!("Starting warm up job");
+        for _ in 0..5 {
+            self.emit_txn_for(Duration::from_secs(20), self.cluster.instances().clone())?;
```

Comment:
Why not just run `emit_txn_for` 100s instead of 20s x 5

Reply:
Good point